### DISCUSSION TITLE
6771ad7b92466f8c6ec38afc

### DISCUSSION THREAD

**Discussion by giscus** [Admin-ToxicGames/toxicgames#10] [General]
# 6771ad7b92466f8c6ec38afc

Discover a wide range of cracked games, mods, and gaming resources for PC, Mac, and more. Download your favorite games for free and enjoy endless gaming adventures!

https://toxicgames.in/6771ad7b92466f8c6ec38afc

**Comment by Admin-ToxicGames** [comment]
<h2>Change Language</h2>

To change the language, edit the CPY.ini file 

Path: /Applications/Far Cry 5.app/Contents/SharedSupport/prefix/drive_c/Program Files (x86)/Far Cry 5/bin/CPY.ini

Replace: Language=ru-RU

On: Language=en-US

**Comment by vinaykrishna7003-rgb** [comment]
will it work on m2 air 8gb ram

**Comment by Ytterbium1998** [comment]
wow wich program you use ?

**Comment by Ytterbium1998** [comment]
works on MacBook Pro m4 16 gb very good

**Comment by Star4999** [comment]
bro it is stucking when I start the game it is showing two bald soldier screen and stucks is there any fix

**Reply by Star4999** [reply]
if any one wants to know how it runs on mac m4 it will run around 30-40 fps

**Reply by Star4999** [reply]
mine is m4 air 15'
